macos/FluentUI/Link.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+/// A lightweight hyperlink control
+@objc(MSFLink)
+open class Link : NSButton {
+	
+	/// Initialies a link with a content string that is displayed, and an underlying
+	/// - Parameters:
+	///   - content: The visible text of the link that the user sees.
+	///   - url: The URL that is opened when the link is clicked
+	@objc public init(content: String, url: NSURL) {
```

Comment:
I'm guessing this might be the control spec, but do we like having this as a URL as opposed to a closure that we call when this is invoked?

Reply:
Not sure I understand what you mean?

Reply:
As it works right now, this is a button that looks like a link. We could scope the control to only work as a link that opens a URL, or we could make it more generic and let clients handle what happens when this button is pressed, even if it's something other than opening a URL.

Reply:
Ah, I specifically wanted to make it only open URL's and nothing else. I figured if clients wanted to do more, they could use the (nonexistent) button control